### ISSUE TITLE
Htop for Windows

### ISSUE THREAD

**Issue by itsme-alan** [microsoft/terminal#8539]
**I don't know if this in scope for this repository. But since this repo houses the command line infrastructure of Windows. I thought I would ask here**

# Description of the new feature/enhancement

Htop is very useful. I would like to see something similar in Windows

# Proposed technical implementation details (optional)

**Comment by zadjii-msft** [comment]
I suppose this is the best place to ask for such a feature. `htop` is a great utility, and I think I end up installing it on every linux box I end up working on. That being said, I doubt that we'll be shipping a version of it for Windows ourselves 😢 I'm not sure that there is a truly good equivalent on Windows, though [ntop](https://github.com/gsass1/NTop) looks like it might be good. Maybe that's a good place for developers to contribute?

**Comment by Don-Vito** [comment]
@zadjii-msft - shouldn't it be somehow based on Sys Internals?

**Comment by itsme-alan** [comment]
@zadjii-msft I meant to ask for a Windows executable that lives in System32 and is shipped with every Windows 10 install

**Comment by itsme-alan** [comment]
I don't want an alternative like ntop

**Comment by zadjii-msft** [comment]
That would maybe make sense. Is Sys Internals still maintained? Wow yes it is. TIL. They've already done the hard work, so it can't be that hard to take the backend they've already got and turn that into a CUI interface, _right_ 😆?

>  a Windows executable that lives in System32 and is shipped with every Windows 10 install

That's pretty unlikely to happen tbh. There's a lot more bureaucracy involved in shipping something in-box rather than just shipping it as an OSS tool that users can download with something like `winget`. Even if it were a first-party tool - `colortool` ships out of this repo, and that's the closest precedent that this team has. Heck, _even the Terminal itself_ isn't in-box 😜

**Comment by itsme-alan** [comment]
At least a Microsoft maintained "htop" would be nice. I don't mind if I had to download it manually

**Comment by DHowett** [comment]
Thanks for the request! You found the right folks to ask, but unfortunately we don't have the capacity or charge to work on this right now. There's a lot of love necessary for our commandline experience to be better, but the best way forward is to get some external developers really into it and see what good they can make.

We're a lot different now than we used to be: we ship bsdtar and curl _inbox_ now, and they're open-source projects other people wrote long before we were here. That doesn't mean we'll bring any random OSS tool inbox, but it does mean that we care about the experience here.

Closing as a "good idea that we don't have time for"
